sys/mips/include/cpufunc.h

```diff
@@ -164,7 +164,9 @@ mips_wr_ ## n(uint64_t a0)					\
 } struct __hack
 
 #if defined(__mips_n64)
+#if !__has_feature(capabilities)
```

Comment:
This might want a comment, perhaps something like 
```
/* While CHERI-MIPS exposes coprocessor 0's excpc register, we do not
 * define the name here.  Instead, we use the capability analogue, ... .
 * This ensures that we use the same code path when dealing with
 * hardware-triggered exceptions and for installing sentry PCCs.
 */
```

Reply:
Makes sense. Alternatively I could use a hardcoded constant instead of the macro.